### ISSUE TITLE
Cannot Access Character Select Screen (Login Timed out)

### ISSUE THREAD

**Issue by technomork** [WarEmu/WarBugs#18071]
Good evening! I hope this bug report finds you well :) I spoke with Sioding about my issue, and he referred me to Wargrimnir. Wargrimnir referred me to the bugtracker page after he tried to help me but could not. I hope you guys can help me out, as I'm suffering from a lack of Waaaghh!! 

**When I log into my test account I can get to character select screen, and log into the game fine. When I try to login to my main account, I cannot access the character select screen. I am stuck in the screen shown in in the attached screenshots**

**Steps to reproduce the problem: Anytime I try to log in from my main account, from multiple machines.
![screenshot_006](https://user-images.githubusercontent.com/74384101/99005161-50c83500-250e-11eb-9c7c-6ba82d146093.jpg)
![screenshot_007](https://user-images.githubusercontent.com/74384101/99005162-5160cb80-250e-11eb-9f41-518a9a9d8eb4.jpg)
![screenshot_008](https://user-images.githubusercontent.com/74384101/99005163-5160cb80-250e-11eb-8237-dbc64a0b1309.jpg)

**

**Testing Screenshots/Videos/Evidences (always needed):**
<!
![Screenshot 2020-11-12 at 5 33 18 PM](https://user-images.githubusercontent.com/74384101/99005291-85d48780-250e-11eb-98bd-40cdb8d9718d.png)
![Screenshot 2020-11-12 at 5 33 44 PM](https://user-images.githubusercontent.com/74384101/99005302-89680e80-250e-11eb-8db6-a207ac70038b.png)
![Screenshot 2020-11-12 at 5 34 02 PM](https://user-images.githubusercontent.com/74384101/99005310-8ec55900-250e-11eb-9503-c025cd3dc445.png)
![Screenshot 2020-11-12 at 5 34 26 PM](https://user-images.githubusercontent.com/74384101/99005315-91c04980-250e-11eb-8f64-2aea68f2cba7.png)
![Screenshot 2020-11-12 at 5 35 26 PM](https://user-images.githubusercontent.com/74384101/99005324-971d9400-250e-11eb-8481-7048bc76beb2.png)

-- Drag and drop an image file here to include it directly in the bug report,
     no need to upload it to another site -->

**Comment by Ysaran** [comment]
I have the very same problem after last hotfix

**Comment by Sixmoons48** [comment]
Been in contact with sullemunk a bit, having same issue here and no clue on fixing it atm it seems

**Comment by technomork** [comment]
Sorry to hear you guys are also having this issue, but I'm also glad I'm not the only one.... really hope they can use our information to find a solution to this. My issues started happening right after the patch following the Halloween event.

**Comment by Sixmoons48** [comment]
Yeah, hopefully if enough people get the issue it might spur them into fixing it for the worry of it happening on a larger scale, it hurts being unable to play my 4 year old account ~.~

**Comment by Hephymetal** [comment]
Yes same Problem here after the last Patch! Was Kinda City time and the restart and after the servers come back online zjo "Login Timed Out".  Never saw this Problem before TBH and i played a lot of War the last Year:) Hope they find the Problem soon <3 Thanks for the Work

**Comment by technomork** [comment]
Hey have any of you tried having a GM move the last toon you logged in, like when they're stuck in zones at loading screen? Wonder if that's been tried yet

**Comment by Sixmoons48** [comment]
Haven't tried asking that, if anyone is willing to ask pls do, it's hard for me to contact gms since they don't exactly pull punches with their passive-aggression when asking for genuine help

**Comment by Sixmoons48** [comment]
i tried loggin in about 3 minutes ago, and was able to get through to my character select! check yalls account, it might just be a christmas present from the devs or a miracle itself :)

**Comment by technomork** [comment]
> i tried loggin in about 3 minutes ago, and was able to get through to my character select! check yalls account, it might just be a christmas present from the devs or a miracle itself :)

No such luck for me :(((
I'm glad you got back into your account though man!!

**Comment by Sixmoons48** [comment]
hmmm, would suggest bug a dev real quick and ask if they did anything specific to my account to fix it, and to try it on yours, i didnt ask so unsure what exactly they did

**Comment by technomork** [comment]
> hmmm, would suggest bug a dev real quick and ask if they did anything specific to my account to fix it, and to try it on yours, i didnt ask so unsure what exactly they did

Will do bud, I assumed Sixmoons48 is your account name yes?

**Comment by technomork** [comment]
Who were you speaking with mostly?

**Comment by AAT-msk** [comment]
same problem after path 01-01-2021
My main account "AAT",   cannot access the character select screen.
Test account,  log into the game fine.

**Comment by Sixmoons48** [comment]
Yeah that's my account name, I spoke mostly with sullemunk, but he wasn't the one that directly fixed it, if issue is till happening to new players on new patches, then doesn't look like it's wholly fixed, just spot fixed from new random patches happening

**Comment by MaxHayman** [comment]
Is this still happening?

**Comment by AAT-msk** [comment]
Yes, Thank you01:52, 23 апреля 2021 г., Max Hayman ***@***.***>:
Is this still happening?

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by technomork** [comment]
> Is this still happening?

My issued was fixed by you back in January Max, thanks so much!

**Comment by TalatCikikci** [comment]
Hello, 

I had the same issue as described in OP though I did not try to see if alternate accounts worked. 

Thing is; I am running the game _**in a VM**_, and changing the network connection from _NAT_ to _Bridged_ in the VM settings worked for me. I doubt this would help a lot of people but if you are running the game in a similar manner and stumbled upon this issue it might be worth trying.
